frontend/isoko/src/pages/Business.tsx

```diff
@@ -89,7 +91,7 @@ const Business: React.FC = () => (
                   reviewerImageUrl="https://www.pluggedin.com/wp-content/uploads/2020/01/bobs-burgers-review-image.jpg"
                   stars={4}
                   subject="Great food and service!"
-                  content="I've never regretted eating at Bill's Diner. The Signature Burger is to die for and Bill is a homie. Everyone should come and eat some of Bill's food made with love and care."
```

Comment:
curious why this was changed

Reply:
On accident lol, I was getting a weird error with the apostrophes at some point, but it was cause I accidentally  commented out a closing apostrophe. I'll go back and fix these as, good catch